app/lib/RouteResolver.js

```diff
@@ -33,13 +33,30 @@ function isRouteForRequest(route, req) {
 
   if (route.pathname !== '*' && !route.pathRegExp.test(pathname)) return false;
 
-  // TODO: Later add features to match other things, like query parameters, etc.
+  const matchesParams = _.every(route.query, (value, key) => {
```

Comment:
I assume you chose `_.every` because `route.query` is an object.

Reply:
Correct.